applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -61,6 +61,10 @@ def __init__(self, Model, settings):
         KratosMultiphysics.FindNodalNeighboursProcess(
             self.fluid_model_part, avg_elem_num, avg_node_num).Execute()
 
+        for cond in self.body_model_part.Conditions:
+            for node in cond.GetNodes():
+                node.Set(KratosMultiphysics.SOLID)
```

Comment:
```suggestion
                node.Set(KratosMultiphysics.STRUCTURE)
```

this is what is ususally used for those purposes

Reply:
I believe @marcnunezc is using SOLID because STRUCTURE is already taken for the trailing edge node. I think it is better if we use our own variables to make clear what is what. For example, instead of SOLID we could use AIRFOIL and instead of STRUCTURE we could use TRAILING_EDGE_NODE, what do you think @marcnunezc ?

Reply:
Hi! 

The reason why I'm using SOLID is because we are already using the flag STRUCTURE (as a placeholder) to mark the element that is both touching the trailing edge and cut by the wake.

I agree this should be changed, but maybe it's better to do so in a separate PR. Do you agree? @inigolcanalejo

Reply:
Ok I understand
I recommend to leave it as is for this PR and change it in a separate one, since there are some technical compliactions that will pop up

Reply:
Sorry I didn't see you answer before submitting mine.

Yes I agree! However, it would be better do the change in a separate PR maybe?